resources/views/landing.blade.php

```diff
@@ -1,3 +1,158 @@
 <x-layouts.app>
-  <h1 class="font-playfair text-3xl text-center">Sākums</h1>
+	
+<div class="landing" x-data="{ scrolled: false }" @scroll="scrolled = ($event.target.scrollTop > 0)">	
+		<button class="back-to-top" x-show="scrolled" x-data="{ scrollToTop: () => { document.querySelector('.landing').scrollTo({ top: 0, behavior: 'smooth' }); } }" @click="scrollToTop()">
+			<x-icons.down></x-icons.down>
+		</button>
+		<section class="landing-section-wrapper">
+			<img class="screen-image" src="{{ asset('storage/landing.webp') }}" alt="">
+			<div class="content">
+				<p>Laipni lūgti</p>
+				<h1 class="hero-title">Ruckas muižā</h1>
+		
+				<div class="user-guidance">
+					<x-icons.down></x-icons.down>
+					<p>ritiniet uz leju</p>
+				</div>
+			</div>
+		</section>	
+		
+		<section class="landing-section-wrapper">
+			<img class="screen-image" src="{{ asset('storage/landing2.webp') }}" alt="">
+			<div class="content">
+				<div class="center-wrapper">
+					<h2>Ko darīt muižā?</h2>
+					<div class="options-grid">
+						<x-landing.option
+							icon="components.icons.confetti"
+							title="Svinēt svētkus"
+							link="svinet-svetkus"
+						/>
+						<x-landing.option
+							icon="components.icons.bedtime"
+							title="Nakšņot"
+							link="naksnot"
+						/>
+						<x-landing.option
+							icon="components.icons.calendar"
+							title="Apmeklēt kultūras pasākumus"
+							link="apmeklet-kulturas-pasakumus"
+						/>
+						<x-landing.option
+							icon="components.icons.fireplace"
+							title="Telpu noma"
+							link="telpu-noma"
+						/>
+						<x-landing.option
+							icon="components.icons.bag"
+							title="Skolas soma"
+							link="skolas-soma"
+						/>
+						<x-landing.option
+							icon="components.icons.bird"
+							title="Iepazīt muižas putnus"
+							link="iepazit-muizas-putnus"
+						/>
+						<x-landing.option
+							icon="components.icons.plant"
+							title="Kļūt par melderi"
+							link="klut-par-melderi"
+						/>
+						<x-landing.option
+							icon="components.icons.forest"
+							title="Aktivitātes, pastaigas muižas parkā"
+							link="aktivitates-pastaigas-muizas-parka"
+						/>
+						<x-landing.option
+							icon="components.icons.brush"
+							title="Dzīvās gleznas"
+							link="dzivas-gleznas"
+						/>
+					</div>
+				</div>
+			</div>
+		</section>
+		<x-landing.section
+			id="svinet-svetkus"
+			bgImage="storage/svinet-svetkus.webp"
+			title="Svinēt svētkus"
+			description="Ruckas muiža piedāvā svinēt kā mazus, tā lielus svētkus! Sākot ar dzimšanas dienām, beidzot ar kāzām."
+    		icon="components.icons.confetti"
+			link="izklaide"
+			right=true
+			fade=true
```

Comment:
Kāpēc dažiem ir fade, citiem nav? Liekas, ka to nevajag

Reply:
Dažām bildēm, kurām fade ir ieslēgts, bilde izskatās labāka, vai ir daudz pārredzamāka.
Kā piemēram, hero section ir fade, jo bez tā "Laipni lūgti Ruckas muižā" teksts būtu nesaredzams.


Tāpēc es pieliku, ka tas ir optional, ka fade ir vai nav.